about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>RMedia</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
```

Comment:
For responsiveness  the  3 sections  - menu , content , blog posts  should wrap below one another on smaller screens. 
Hint : use media queries to set breakpoints

Reply:
done